### ISSUE TITLE
Miscellaneous questions

### ISSUE THREAD

**Issue by Niagara1000** [DS4PS/cpp-528-spr-2021#52]
Hi @cenuno ,

I have some more questions:

1. Can we delete the `labs/wk07/` folder since we do not have a Lab 7?
2. Just wanted to verify: do we close issues when they are done or is there a step in between 1. moving it to `Done` and 2. closing them?
3. Where is the `ltdb-data-dictionary.csv` supposed to live? on different parts of the course schedule, it says different locations, so I wanted to make sure with you first
4. Are the following 4 folders shown below containing the correct files?

<br>

**pic 1**
<img width="817" alt="Screen Shot 2021-04-18 at 3 46 20 PM" src="https://user-images.githubusercontent.com/46457863/115174259-c87e2b00-a07d-11eb-9168-e9eb1ae1ad42.png">

<hr><br>

**pic 2**
<img width="811" alt="Screen Shot 2021-04-18 at 3 46 40 PM" src="https://user-images.githubusercontent.com/46457863/115174275-ce740c00-a07d-11eb-8188-1afc90c00897.png">

<hr><br>

**pic 3**
<img width="804" alt="Screen Shot 2021-04-18 at 3 46 47 PM" src="https://user-images.githubusercontent.com/46457863/115174287-d2a02980-a07d-11eb-977f-4408351fa08c.png">

<hr><br><br>

I have a feeling that some files that were supposed to be in the `wrangling/` folder are placed in other `data/` folders. What are the other files we need to include under `wrangling/` ?

**pic 4**
<img width="813" alt="Screen Shot 2021-04-18 at 3 46 58 PM" src="https://user-images.githubusercontent.com/46457863/115174294-d7fd7400-a07d-11eb-968d-94cf7fb8ae00.png">

<hr><br>

**Comment by cenuno** [comment]
> Hi @cenuno ,
> 
> I have some more questions:
> 
> 1. Can we delete the `labs/wk07/` folder since we do not have a Lab 7?

Yes, please do. I'll be sure to delete this from the rubric for the next course.

> 2. Just wanted to verify: do we close issues when they are done or is there a step in between 1. moving it to `Done` and 2. closing them?

The 'DOING' part is mostly captured when you tag your issues within your pull requests (see #53 for an example). As long as you have your automations set up, the task will travel automatically from TO DO, to DOING, to finally DONE when the pull request is merged back into `main`.

> 3. Where is the `ltdb-data-dictionary.csv` supposed to live? on different parts of the course schedule, it says different locations, so I wanted to make sure with you first

Good catch, I think it would make sense for this to go into `data/rodeo/`. Thanks for flagging.

> 4. Are the following 4 folders shown below containing the correct files?
> 
> 
> **pic 1**
> <img alt="Screen Shot 2021-04-18 at 3 46 20 PM" width="817" src="https://user-images.githubusercontent.com/46457863/115174259-c87e2b00-a07d-11eb-9168-e9eb1ae1ad42.png">
> 
> 
> **pic 2**
> <img alt="Screen Shot 2021-04-18 at 3 46 40 PM" width="811" src="https://user-images.githubusercontent.com/46457863/115174275-ce740c00-a07d-11eb-8188-1afc90c00897.png">
> 
> 
> **pic 3**
> <img alt="Screen Shot 2021-04-18 at 3 46 47 PM" width="804" src="https://user-images.githubusercontent.com/46457863/115174287-d2a02980-a07d-11eb-977f-4408351fa08c.png">
> 
> 
> 
> I have a feeling that some files that were supposed to be in the `wrangling/` folder are placed in other `data/` folders. What are the other files we need to include under `wrangling/` ?
> 
> **pic 4**
> <img alt="Screen Shot 2021-04-18 at 3 46 58 PM" width="813" src="https://user-images.githubusercontent.com/46457863/115174294-d7fd7400-a07d-11eb-968d-94cf7fb8ae00.png">

Yes, all the files that should be there are there with one exception: there should not be `README.html` files. Please delete those files.

As far as `data/wrangling/` is concerned, I am not surprised to see nothing in there. I didn't use in my tutorials or ask that it be used. I'll revisit this when planning next semester's course.

**Comment by Niagara1000** [comment]
Thank you for your answers!
